app/src/main/res/layout/fragment_suggested_edits_cards_item.xml

```diff
@@ -40,143 +36,157 @@
             android:background="?attr/button_background_color"
             android:visibility="gone" />
 
-        <LinearLayout
+        <androidx.core.widget.NestedScrollView
             android:id="@+id/cardItemContainer"
             android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:orientation="vertical">
-
-            <org.wikipedia.views.FaceAndColorDetectImageView
-                android:id="@+id/viewArticleImage"
-                style="@style/SimpleDraweeViewPlaceholder.SuggestedEdits"
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                app:actualImageScaleType="focusCrop"
-                app:viewAspectRatio="1.77"
-                app:fadeDuration="0" />
+            android:layout_height="wrap_content">
 
             <LinearLayout
-                android:id="@+id/viewArticleContainer"
+                android:id="@+id/cardClickArea"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:minHeight="231dp"
-                android:orientation="vertical"
-                android:paddingTop="11dp">
-
-                <TextView
-                    android:id="@+id/viewArticleTitle"
+                android:clickable="true"
+                android:focusable="true"
+                android:background="?attr/selectableItemBackground"
```

Comment:
It would be better if change this to `android:foreground="?attr/selectableItemBackground"`

Reply:
Why is that?

Reply:
The `ImageView` and the `LinearLayout` that contains a blue left label with a description will not be covered by ripple animation.

Reply:
Ah ok. I was testing on API 27, where the issue wasn't as evident.
Done.